### ISSUE TITLE
v2.11.0 Uncaught TypeError when DD_TRACE_ENABLED=false

### ISSUE THREAD

**Issue by arminm** [DataDog/dd-trace-js#2222]
I just updated our dd-trace package from 1.3.0 to 2.11.0 and now when running integration tests in our project, we set DD_TRACE_ENABLED=false, and get this error:
```
Uncaught TypeError: Cannot read properties of undefined (reading '_tags')
  at addError (/.../node_modules/dd-trace/packages/dd-trace/src/plugins/plugin.js:56:34)
  @service-test:       at Subscription._handler (/.../node_modules/dd-trace/packages/dd-trace/src/plugins/plugin.js:12:9)
  @service-test:       at Channel.publish (node:diagnostics_channel:56:9)
  @service-test:       at /.../node_modules/dd-trace/packages/datadog-instrumentations/src/router.js:94:22
...
```
previously we would set `enabled: false`. It seems like setting the ENV var doesn't completely disable the tracer. Is this a bug? or is there a better way to disable the tracer for testing purposes?

**Comment by raxityo** [comment]
I also just noticed this a few minutes ago, @arminm. Strange co-incidence 🙂
Looks like this was reported a week ago in #2201 and fixed in #2209, which isn't out yet.

**Comment by arminm** [comment]
hah! thanks for linking the fix. I couldn't find that issue in my search 😓 Should have searched the closed issues as well.
Ii guess i'll wait for the next release. Downgraded to 2.10.0 for now and it works 👍
